### ISSUE TITLE
Non Existent Envoirnments in Conf.js should return error instead of returning the default envoirnment settings

### ISSUE THREAD

**Issue by ajasingh** [single-spa/import-map-deployer#96]
I have configured two envoirnment in my conf.js with below file which are working fine when i get and patch import-map.json but it always returns sit1 file for wrong envoirnment like sit3 and sit4 which are not even existing in conf.js , ideally it should throw error if wrong envoirnment name is passed in the query string

Below url still works which should not work 
**import-map.json?env=sit3**
```
module.exports = {
manifestFormat: "importmap",
packagesViaTrailingSlashes: false,
locations: {
sit1: "/www/mfe-sit/sit1/microfrontends/import-map.json",
sit2: "/www/mfe-sit/sit2/microfrontends/import-map.json",
},
};
```

**Comment by joeldenning** [comment]
This was introduced in https://github.com/single-spa/import-map-deployer/pull/95 (cc @vongohren).

The intent of #95 was to use the first location when the `env` was omitted entirely (as happens when doing the initial health check). However, it seems it's using the first location even when `env` is present.

I'll put together a fix.

**Comment by joeldenning** [comment]
I've fixed this in #97

**Comment by joeldenning** [comment]
This is released in https://github.com/single-spa/import-map-deployer/releases/tag/v4.0.0
